examples/apps/eea_token/readme

```diff
@@ -0,0 +1,69 @@
+
+EEA token execution logic allows running the execution logic based on token requests,
+alse see check https://github.com/EntEthAlliance/Trusted-Token token_validation.cpp
```

Comment:
The link https://github.com/EntEthAlliance/Trusted-Token is broken. Is there any link where we can find more details about trusted token and about functionality implemented in this PR

Reply:
Sanjay Bakshi may have a link.
I do not know of any.

Reply:
This link is right and the github project is currently private

Reply:
I confirmed the link is correct with Sanjay.